dsl.md

```diff
@@ -113,7 +113,7 @@ The Serverless Workflow DSL defines several default [task](dsl-reference.md#task
 - [Try](dsl-reference.md#try), used to attempt executing a specified [task](dsl-reference.md#task), and to handle any resulting [errors](dsl-reference.md#error) gracefully, allowing the [workflow](dsl-reference.md#workflow) to continue without interruption.
 - [Wait](dsl-reference.md#wait), used to pause or wait for a specified duration before proceeding to the next task.
 
-To ensure they conform to the DSL, runtimes **should** pass all the feature conformance test scenarii defined in the [ctk](ctk/README.md).
```

Comment:
afaik `scenarii` is the correct plural form of `scenario`, no?

Reply:
Indeed, but quite ols. I think `scenarios` fits better in this case:

> (nonstandard, rare)

https://www.yourdictionary.com/scenarii

Reply:
Also this guy here gave more background: https://english.stackexchange.com/questions/11775/what-is-the-plural-of-scenario

Sorry to spam to much messages, I was curious about the `scenarii` never knew!! I learned something new today, hahaha

Reply:
@cdavernas Actually, you are right lol. Sometime, I use scenarios, but I think scenarii is good. I will change back.